Tests/configure_and_test_integration_instances.py

```diff
@@ -679,11 +723,11 @@ def main():
 
     if new_integrations_files:
         # TODO: uncomment when we start testing packs
-        # if server_version_compare(server_numeric_version, '6.0') >= 0:
-        #    # Test packs search and installation - beginning of infrastructure
-        #    client = demisto_client.configure(base_url=servers[0], username=username, password=password,
-        #                                      verify_ssl=False)
-        #    search_and_install_packs_and_their_dependencies(new_integrations_files, client, prints_manager)
+        if server_version_compare(server_numeric_version, '6.0') >= 0:
```

Comment:
Could you just use LooseVersion instead? this kind of logic already made couple of bugs in the build

Reply:
i just commented out the code here, as it was waiting for changes done in this pull request
didnt write the code, so prefer not to change it in this PR